models/codegen.config

```diff
@@ -35,10 +35,6 @@ YANGAPI_EXCLUDES +=
 # over this list. Note that the entry should be the yang module name
 # which is used for generated yaml file name.
 PY_YANGAPI_CLIENTS += 
```

Comment:
As said in the PR description, the change is intended to discontinue swagger client support. 
Can you remove the variables to be deprecated in the codegen.config file?

Reply:
I expect these cleanups to be part of openapi generator upgrade PR, which will come next. Purpose of this PR is to migrate existing cli handlers only.